### PR TITLE
task solution 1

### PR DESCRIPTION
**Pull request by pr1syazhniy**
[DEMO LINK](https://pr1syazhniy.github.io/layout_product-cards/)
[TEST REPORT LINK](https://pr1syazhniy.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/cardproduct.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+html {
```
Comment (slavanosatskyi) [inline src/cardproduct.css:1]: It should be defined in another file (style.css for example)

File: src/cardproduct.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card-product__code {
+  height: 14px;
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
```
Comment (slavanosatskyi) [inline src/cardproduct.css:47]: and this

File: src/cardproduct.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  font-style: normal;
+}
+
+.card-product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px ;
```
Comment (ivanchuk-yurii) [inline src/cardproduct.css:12]:
```suggestion
  padding: 32px 16px 16px;
```

File: src/cardproduct.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__name {
+  margin-top: 40px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
```
[the remark this answers is not in the document]
Comment (ivanchuk-yurii) [reply src/cardproduct.css:33]: not fixed, move all colors into variables

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,54 @@
...
+          </div>
+          <div class="card-product__reviews_rate">Reviews: 5</div>
+        </div>
+
+        <div class="card-product__price">
+          <div class="card-product__price_title">Price:</div>
+          <div class="card-product__price_money">$2,199</div>
+        </div>
+
+        <div class="card-product__buy" data-qa="hover">
```
Comment (ivanchuk-yurii) [inline src/index.html:43]: this `div` is redundant

File: src/reset.css [hunk 6]
```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  padding: 0;
```
Comment (ivanchuk-yurii) [inline src/reset.css:3]: `body` doesn't have default `padding`

File: src/stars.css [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (ivanchuk-yurii) [inline src/stars.css:18]: combine into one `background` property

File: src/stars.css [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.stars_star:nth-child(-n + 4) {
```
Comment (ivanchuk-yurii) [inline src/stars.css:21]: should also work for other number of stars

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,9 @@
+@import "./reset.css";
+
+@import "./cardproduct.css";
+
+@import "./stars.css";
```
Comment (ivanchuk-yurii) [inline src/style.css:5]:
```suggestion
@import "./reset.css";
@import "./cardproduct.css";
@import "./stars.css";
```

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 5 above)
Comment (pr1syazhniy) [reply src/index.html:43]: why?

File: src/stars.css (hunk 8 above)
Comment (nazarmatsevych) [reply src/stars.css:20]: * not fixed

File: src/cardproduct.css [hunk 10]
```diff
@@ -0,0 +1,101 @@
+
+* {
+  box-sizing: border-box;
+}
```
Comment (nazarmatsevych) [inline src/cardproduct.css:4]: You can also move it to style.css file

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Try to fix all of the previous comments before making re-request,
if you have any questions - feel free to ask them in fe_chat

File: src/cardproduct.css [hunk 11]
```diff
@@ -0,0 +1,96 @@
+.card-container {
+  display: flex;
+  font-style: normal;
+}
+
+.card-product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 400px;
```
Comment (vlad-demchuk) [inline src/cardproduct.css:10]: do not hardcode height
let content dictate this value

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
looks better, just one thing left to fix

**Review by slavanosatskyi — APPROVED** [review]

**Pull request closed without merging**